renku_notebooks/api/notebooks.py

```diff
@@ -70,28 +90,42 @@ def user_server(user, server_name):
 
 
 @bp.route("servers", methods=["POST"])
+@validate_response_with(
```

Comment:
I want to validate the response because this way there is zero chance that I can send something to the frontend that breaks the schema. Without this (or by using `marshal_with` from `flask-apispec` I can send a response that completely violates the response schema and nothing would happen.